pkg/graph/schema.graphql

```diff
@@ -1499,6 +1517,40 @@ input TRBRequestChanges @goModel(model: "map[string]interface{}") {
     status: TRBRequestStatus 
 }
 
+"""
+Represents an EUA user who is included as an attendee for a TRB request
+"""
+type TRBRequestAttendee {
+  id: UUID!
+  euaUserId: String!
+  trbRequestId: UUID!
+  component: String!
+  role: PersonRole!
+  createdBy: String!
+  createdAt: Time!
+  modifiedBy: String
+  modifiedAt: Time
+}
+
+"""
+The data needed add a TRB request attendee to a TRB request
+"""
+input CreateTRBRequestAttendeeInput {
+  euaUserId: String!
+  trbRequestId: UUID!
+  component: String!
+  role: String!
```

Comment:
Now that you added the `PersonRole`,  I think it would make since to use that here instead of string. That way we remove the risk of trying to write a value to the database that not included in the person role enum. 

Go will allow any string to be cast to the `PersonRole` type, since it is just a type alias for string, but it should fail on the DB level. This would remove this type cast
`Role:         models.PersonRole(input.Role),`

Reply:
![image](https://user-images.githubusercontent.com/99294987/191805160-2eb780f2-1aa2-41d3-9c9e-4917ac3eb906.png)